Makefile

```diff
@@ -119,215 +153,49 @@ benchmarks: $(BENCHSTAT) validate_go ## Benchmark
 run: build ## Run
 	./"${FLP_BIN_FILE}"
 
-##@ Docker
-
-# note: to build and push custom image tag use: DOCKER_TAG=test make push-image
-.PHONY: build-image
-build-image:
-	DOCKER_BUILDKIT=1 $(OCI_RUNTIME) build -t $(DOCKER_IMG):$(DOCKER_TAG) -f contrib/docker/Dockerfile .
-
-# It would have been better to have
-build-single-multiarch-linux/%:
-#The --load option is ignored by podman but required for docker
-	DOCKER_BUILDKIT=1 $(OCI_RUNTIME) buildx build --load --build-arg TARGETPLATFORM=linux/$* --build-arg TARGETARCH=$* --build-arg BUILDPLATFORM=linux/amd64 -t $(DOCKER_IMG):$(DOCKER_TAG)-$* -f contrib/docker/Dockerfile .
-
-# It would have been better to have
-push-single-multiarch-linux/%: build-single-multiarch-linux/%
-#The --load option is ignored by podman but required for docker
-	DOCKER_BUILDKIT=1 $(OCI_RUNTIME) push $(DOCKER_IMG):$(DOCKER_TAG)-$*
-
-# note: to build and push custom image tag use: DOCKER_TAG=test make push-image
-.PHONY: build-multiarch-manifest
-build-multiarch-manifest: push-single-multiarch-linux/amd64 push-single-multiarch-linux/arm64 push-single-multiarch-linux/ppc64le
-	#if using Docker, image needs to be pushed before beeing added to the manifest
-	DOCKER_BUILDKIT=1 $(OCI_RUNTIME) manifest create $(DOCKER_IMG):$(DOCKER_TAG) --amend $(DOCKER_IMG):$(DOCKER_TAG)-amd64 --amend $(DOCKER_IMG):$(DOCKER_TAG)-arm64 --amend $(DOCKER_IMG):$(DOCKER_TAG)-ppc64le
-
-.PHONY: push-multiarch-manifest
-push-multiarch-manifest: build-multiarch-manifest
-	@echo 'publish manifest $(DOCKER_TAG) to $(DOCKER_IMG)'
-ifeq (${OCI_RUNTIME} , docker)
-		DOCKER_BUILDKIT=1 $(OCI_RUNTIME) manifest push $(DOCKER_IMG):$(DOCKER_TAG)
+##@ Images
+
+# note: to build and push custom image tag use: IMAGE_ORG=myuser VERSION=dev make images
+.PHONY: image-build
+image-build: ## Build MULTIARCH_TARGETS images
+	trap 'exit' INT; \
+	$(foreach target,$(MULTIARCH_TARGETS),$(call build_target,$(target)))
+
+.PHONY: image-push
+image-push: ## Push MULTIARCH_TARGETS images
+	trap 'exit' INT; \
+	$(foreach target,$(MULTIARCH_TARGETS),$(call push_target,$(target)))
```

Comment:
how long it takes before and after this change to build and push images 
I am curious if that takes much longer time we could have multi-target build option and just leave the current options for the default target ?
depends on how frequently we need to build for all targets do we do that when release new version or on every PR ?

Reply:
It's mainly 3 times longer as `MULTIARCH_TARGETS` contains `amd64 arm64 ppc64le` currently. The manifest creation is super fast and can even be skipped if you point a single arch image.
I'm fine setting `amd64` only as default and letting a comment with example to build the 3 archs.